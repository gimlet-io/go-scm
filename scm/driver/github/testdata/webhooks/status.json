{
  "id": 10184618567,
  "sha": "00e1ed7b5a49e70d5f241768e52b32add7148ff0",
  "name": "laszlocph/demo-app",
  "target_url": "https://github.com/laszlocph/gitops-staging/commit/1df82168deeeaa6516668a1c679b8a2ddd019805",
  "avatar_url": "https://avatars1.githubusercontent.com/u/4289031?v=4",
  "context": "gitops/staging/template-update-jul-13-10-58-27",
  "description": "[Gimlet] 00e1ed7@laszlocph/demo-app/staging - GitOps update due to adhoc deploy",
  "state": "success",
  "commit": {
    "sha": "00e1ed7b5a49e70d5f241768e52b32add7148ff0",
    "node_id": "MDY6Q29tbWl0MjQwMzI3NjE0OjAwZTFlZDdiNWE0OWU3MGQ1ZjI0MTc2OGU1MmIzMmFkZDcxNDhmZjA=",
    "commit": {
      "author": {
        "name": "Laszlo Fogas",
        "email": "xxx",
        "date": "2020-07-06T10:51:53Z"
      },
      "committer": {
        "name": "Laszlo Fogas",
        "email": "xxx",
        "date": "2020-07-06T10:51:53Z"
      },
      "message": "Using db config",
      "tree": {
        "sha": "c666552d583229041b9642c70d8626db075c81ad",
        "url": "https://api.github.com/repos/laszlocph/demo-app/git/trees/c666552d583229041b9642c70d8626db075c81ad"
      },
      "url": "https://api.github.com/repos/laszlocph/demo-app/git/commits/00e1ed7b5a49e70d5f241768e52b32add7148ff0",
      "comment_count": 0,
      "verification": {
        "verified": false,
        "reason": "unsigned",
        "signature": null,
        "payload": null
      }
    },
    "url": "https://api.github.com/repos/laszlocph/demo-app/commits/00e1ed7b5a49e70d5f241768e52b32add7148ff0",
    "html_url": "https://github.com/laszlocph/demo-app/commit/00e1ed7b5a49e70d5f241768e52b32add7148ff0",
    "comments_url": "https://api.github.com/repos/laszlocph/demo-app/commits/00e1ed7b5a49e70d5f241768e52b32add7148ff0/comments",
    "author": {
      "login": "laszlocph",
      "id": 4289031,
      "node_id": "MDQ6VXNlcjQyODkwMzE=",
      "avatar_url": "https://avatars1.githubusercontent.com/u/4289031?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/laszlocph",
      "html_url": "https://github.com/laszlocph",
      "followers_url": "https://api.github.com/users/laszlocph/followers",
      "following_url": "https://api.github.com/users/laszlocph/following{/other_user}",
      "gists_url": "https://api.github.com/users/laszlocph/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/laszlocph/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/laszlocph/subscriptions",
      "organizations_url": "https://api.github.com/users/laszlocph/orgs",
      "repos_url": "https://api.github.com/users/laszlocph/repos",
      "events_url": "https://api.github.com/users/laszlocph/events{/privacy}",
      "received_events_url": "https://api.github.com/users/laszlocph/received_events",
      "type": "User",
      "site_admin": false
    },
    "committer": {
      "login": "laszlocph",
      "id": 4289031,
      "node_id": "MDQ6VXNlcjQyODkwMzE=",
      "avatar_url": "https://avatars1.githubusercontent.com/u/4289031?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/laszlocph",
      "html_url": "https://github.com/laszlocph",
      "followers_url": "https://api.github.com/users/laszlocph/followers",
      "following_url": "https://api.github.com/users/laszlocph/following{/other_user}",
      "gists_url": "https://api.github.com/users/laszlocph/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/laszlocph/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/laszlocph/subscriptions",
      "organizations_url": "https://api.github.com/users/laszlocph/orgs",
      "repos_url": "https://api.github.com/users/laszlocph/repos",
      "events_url": "https://api.github.com/users/laszlocph/events{/privacy}",
      "received_events_url": "https://api.github.com/users/laszlocph/received_events",
      "type": "User",
      "site_admin": false
    },
    "parents": [
      {
        "sha": "aa2fba0052bd2409f413b9202bb0805b10cca4f6",
        "url": "https://api.github.com/repos/laszlocph/demo-app/commits/aa2fba0052bd2409f413b9202bb0805b10cca4f6",
        "html_url": "https://github.com/laszlocph/demo-app/commit/aa2fba0052bd2409f413b9202bb0805b10cca4f6"
      }
    ]
  },
  "branches": [
    {
      "name": "master",
      "commit": {
        "sha": "42e69de45fb64b4e02f4556071f488496a8e033e",
        "url": "https://api.github.com/repos/laszlocph/demo-app/commits/42e69de45fb64b4e02f4556071f488496a8e033e"
      },
      "protected": false
    }
  ],
  "created_at": "2020-07-13T10:58:32+00:00",
  "updated_at": "2020-07-13T10:58:32+00:00",
  "repository": {
    "id": 240327614,
    "node_id": "MDEwOlJlcG9zaXRvcnkyNDAzMjc2MTQ=",
    "name": "demo-app",
    "full_name": "laszlocph/demo-app",
    "private": true,
    "owner": {
      "login": "laszlocph",
      "id": 4289031,
      "node_id": "MDQ6VXNlcjQyODkwMzE=",
      "avatar_url": "https://avatars1.githubusercontent.com/u/4289031?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/laszlocph",
      "html_url": "https://github.com/laszlocph",
      "followers_url": "https://api.github.com/users/laszlocph/followers",
      "following_url": "https://api.github.com/users/laszlocph/following{/other_user}",
      "gists_url": "https://api.github.com/users/laszlocph/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/laszlocph/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/laszlocph/subscriptions",
      "organizations_url": "https://api.github.com/users/laszlocph/orgs",
      "repos_url": "https://api.github.com/users/laszlocph/repos",
      "events_url": "https://api.github.com/users/laszlocph/events{/privacy}",
      "received_events_url": "https://api.github.com/users/laszlocph/received_events",
      "type": "User",
      "site_admin": false
    },
    "html_url": "https://github.com/laszlocph/demo-app",
    "description": null,
    "fork": false,
    "url": "https://api.github.com/repos/laszlocph/demo-app",
    "forks_url": "https://api.github.com/repos/laszlocph/demo-app/forks",
    "keys_url": "https://api.github.com/repos/laszlocph/demo-app/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/laszlocph/demo-app/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/laszlocph/demo-app/teams",
    "hooks_url": "https://api.github.com/repos/laszlocph/demo-app/hooks",
    "issue_events_url": "https://api.github.com/repos/laszlocph/demo-app/issues/events{/number}",
    "events_url": "https://api.github.com/repos/laszlocph/demo-app/events",
    "assignees_url": "https://api.github.com/repos/laszlocph/demo-app/assignees{/user}",
    "branches_url": "https://api.github.com/repos/laszlocph/demo-app/branches{/branch}",
    "tags_url": "https://api.github.com/repos/laszlocph/demo-app/tags",
    "blobs_url": "https://api.github.com/repos/laszlocph/demo-app/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/laszlocph/demo-app/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/laszlocph/demo-app/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/laszlocph/demo-app/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/laszlocph/demo-app/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/laszlocph/demo-app/languages",
    "stargazers_url": "https://api.github.com/repos/laszlocph/demo-app/stargazers",
    "contributors_url": "https://api.github.com/repos/laszlocph/demo-app/contributors",
    "subscribers_url": "https://api.github.com/repos/laszlocph/demo-app/subscribers",
    "subscription_url": "https://api.github.com/repos/laszlocph/demo-app/subscription",
    "commits_url": "https://api.github.com/repos/laszlocph/demo-app/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/laszlocph/demo-app/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/laszlocph/demo-app/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/laszlocph/demo-app/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/laszlocph/demo-app/contents/{+path}",
    "compare_url": "https://api.github.com/repos/laszlocph/demo-app/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/laszlocph/demo-app/merges",
    "archive_url": "https://api.github.com/repos/laszlocph/demo-app/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/laszlocph/demo-app/downloads",
    "issues_url": "https://api.github.com/repos/laszlocph/demo-app/issues{/number}",
    "pulls_url": "https://api.github.com/repos/laszlocph/demo-app/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/laszlocph/demo-app/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/laszlocph/demo-app/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/laszlocph/demo-app/labels{/name}",
    "releases_url": "https://api.github.com/repos/laszlocph/demo-app/releases{/id}",
    "deployments_url": "https://api.github.com/repos/laszlocph/demo-app/deployments",
    "created_at": "2020-02-13T18:05:39Z",
    "updated_at": "2020-07-06T11:10:22Z",
    "pushed_at": "2020-07-06T12:59:20Z",
    "git_url": "git://github.com/laszlocph/demo-app.git",
    "ssh_url": "git@github.com:laszlocph/demo-app.git",
    "clone_url": "https://github.com/laszlocph/demo-app.git",
    "svn_url": "https://github.com/laszlocph/demo-app",
    "homepage": null,
    "size": 8,
    "stargazers_count": 0,
    "watchers_count": 0,
    "language": null,
    "has_issues": true,
    "has_projects": true,
    "has_downloads": true,
    "has_wiki": true,
    "has_pages": false,
    "forks_count": 0,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 0,
    "license": null,
    "forks": 0,
    "open_issues": 0,
    "watchers": 0,
    "default_branch": "master"
  },
  "sender": {
    "login": "gimlet-for-staging[bot]",
    "id": 61659236,
    "node_id": "MDM6Qm90NjE2NTkyMzY=",
    "avatar_url": "https://avatars1.githubusercontent.com/u/4289031?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D",
    "html_url": "https://github.com/apps/gimlet-for-staging",
    "followers_url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D/followers",
    "following_url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D/following{/other_user}",
    "gists_url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D/subscriptions",
    "organizations_url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D/orgs",
    "repos_url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D/repos",
    "events_url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D/events{/privacy}",
    "received_events_url": "https://api.github.com/users/gimlet-for-staging%5Bbot%5D/received_events",
    "type": "Bot",
    "site_admin": false
  }
}