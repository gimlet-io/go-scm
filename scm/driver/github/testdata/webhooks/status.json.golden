{
  "SHA": "00e1ed7b5a49e70d5f241768e52b32add7148ff0",
  "Status": {
    "State": 3,
    "Label": "gitops/staging/template-update-jul-13-10-58-27",
    "Desc": "[Gimlet] 00e1ed7@laszlocph/demo-app/staging - GitOps update due to adhoc deploy",
    "Target": "https://github.com/laszlocph/gitops-staging/commit/1df82168deeeaa6516668a1c679b8a2ddd019805",
    "Title": ""
  },
  "Repo": {
    "ID": "240327614",
    "Namespace": "laszlocph",
    "Name": "demo-app",
    "Perm": null,
    "Branch": "master",
    "Private": true,
    "Clone": "https://github.com/laszlocph/demo-app.git",
    "CloneSSH": "git@github.com:laszlocph/demo-app.git",
    "Link": "https://github.com/laszlocph/demo-app",
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
  }
}
